library/core/src/option.rs

```diff
@@ -809,28 +768,29 @@ impl<T> Option<T> {
     #[must_use]
```

Comment:
Not really relevant to this PR, but it's surprising to me that this is marked as `must_use`.